testing.sagebrush.master/Stepwat_Inputs/Input/model.in

```diff
@@ -20,4 +20,4 @@
 
 # niter nyrs  seed
 # 100 300 0 # recommended settings for production runs
-1 300 7 # settings for testing purposes
+3 300 7 # settings for testing purposes
```

Comment:
This needs to be changed back to 1 iteration.

Reply:
@chaukap still need to change this back.

Reply:
Resolved by commit 537b222